CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.1.1 (2020-09-25)
+
+- Fixed the avatar component to properly set accessibility labels and text for screen reader users. [#44](https://github.com/blackbaud/skyux-avatar/pull/44)
```

Comment:
Maybe just say `screen readers` ? Not sure if we already have an established pattern for that...

Reply:
I’m fine with that. Changed.